cdxev/__main__.py

```diff
@@ -625,9 +625,10 @@ def create_build_public_bom_parser(
         type=Path,
     )
     parser.add_argument(
-        "schema_path",
+        "--schema-path",
         metavar="<schema path>",
         help="Path to a json schema, defining when a sbom is considered internal",
```

Comment:
Is this really the correct description? Imo the right wording would be "when **the information** in an SBOM is considered internal".

Reply:
Adopted